recipes/open62541/all/conanfile.py

```diff
@@ -396,24 +422,35 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "open62541"
         self.cpp_info.names["cmake_find_package_multi"] = "open62541"
         self.cpp_info.names["pkg_config"] = "open62541"
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.libs = collect_libs(self)
         self.cpp_info.includedirs = [
             "include",
             os.path.join("include", "open62541", "plugin")
         ]
 
         # required for creating custom servers from ua-nodeset
-        self.user_info.tools_dir = os.path.join(self.package_folder, "res", "tools")
-        self._chmod_plus_x(os.path.join(self.package_folder, "res", "tools", "generate_nodeid_header.py"))
+        self.conf_info.define("user.myconf:tools_dir", os.path.join(
```

Comment:
this conf name is too generic, recipeName should be mentioned, there are some guidelines in conancenter documentation.

Reply:
Could you link the guidelines for that? 

I can't find it in [How To Add Packages](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) or in [Developing Recipes](https://github.com/conan-io/conan-center-index/blob/master/docs/developing_recipes_locally.md)

Reply:
There was a guideline, @prince-chrismc where is it know?

Reply:
@memsharded  suggested `user.open62541:tools_dir` would it work, @SpaceIm?

Reply:
yes, recommended name is `user:<recipeName>:<param>`

Reply:
great, thanks for the response, @SpaceIm ! It would also be really useful if this information was in [How To Add Packages](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) or in [Developing Recipes](https://github.com/conan-io/conan-center-index/blob/master/docs/developing_recipes_locally.md). 

Should I open a separate issue requesting this to be documented in conan-center-index?

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/v2_migration.md#new-conf_info-properties

Reply:
#13673 :) I'm on vacation this weekend 🪐 ✈️  but tag me if you have docs issues and I'll see them when I get back home Tuesday

Reply:
thanks for the info @prince-chrismc !